### PR TITLE
docs: `bebin` functions explained, recursive bebin and more 

### PR DESCRIPTION
**Pull request by iamvee**

### CODE REVIEW

File: docs/_posts/2019-12-25-functions.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title:  "تعریف تابع"
+date:   2019-12-15 00:55:22 +0100
+categories: jekyll update
+permalink: /bebin/
+---
+تابع در زبان برنامه‌نویسی آقای هاشمی با `bebin` تعریف می‌شوند.
```
Comment (amaralani) [inline docs/_posts/2019-12-25-functions.md]: توابع

File: docs/_posts/2019-12-25-functions.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+پس از کلمه‌کلیدی `bebin` نام تابع، در اینجا `esme_functione_bebin` نوشته شده است. 
+سپس ورودی‌های مورد نظر برای تابع را داخل پرانتز مشخص می‌کنیم. 
+توجه داشته باشید که مقادیری که درون پرانتز تعریف شده است، برای اسکوپ داخلی تابع، قابل دسترسی هستند و اصطلاحا local هستند. 
+
+{% highlight java %}
+bebin esme_functione_bebin(voroodi) {
+    khorooji = voroodi; 
+    //    دستورهای مورد نظر برای این تابع تا 
+    //    پیش از رسیدن به دستور «بده» نوشته می‌شود
+    bede khorooji;
```
Comment (amaralani) [inline docs/_posts/2019-12-25-functions.md]: bede :+1: :smile:

File: docs/_posts/2019-12-25-functions.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+}
+{% endhighlight %} 
+
+تعریف تابع، مشابه چیزی که در بالا مشاهده می‌کنید،‌به خودی خود قرار نیست نتیجه‌ای در بر داشته باشد. بنابرین، می‌بایست تابع مورد نظر «صدا زده شود» یا «call» بشود.
```
Comment (amaralani) [inline docs/_posts/2019-12-25-functions.md]: بنابراین

File: docs/_posts/2019-12-25-functions.md (hunk 1 above)
Comment (psycho-ir) [inline docs/_posts/2019-12-25-functions.md]:
```suggestion
توابع در زبان برنامه‌نویسی آقای هاشمی با `bebin` تعریف می‌شوند.
```

**Pull request merged**